comms/schema/schema.ts

```diff
@@ -44,7 +44,7 @@ export type ChatReactRPC = {
   params: {
     chat_id: string
     message_id: string
-    reaction: string
+    reaction?: string
```

Comment:
I think we want explicitly `null`, not allowing `undefined`. wdyt?

Reply:
sure, added in https://github.com/AudiusProject/audius-protocol/pull/4606/commits/8800a10f703f86468d12b0a74bb76856cd8b203a